test/xpu_api/random/interface_tests/distributions_methods.pass.cpp

```diff
@@ -20,10 +20,12 @@
 #include "support/utils.h"
 
 #if TEST_UNNAMED_LAMBDAS
+#include <oneapi/dpl/random>
+static_assert(ONEDPL_HAS_RANDOM_NUMBERS >= 202409L);
```

Comment:
Why this `static_assert` has placed under `#if TEST_UNNAMED_LAMBDAS` check?

Reply:
Probably we should check `ONEDPL_HAS_RANDOM_NUMBERS` and `ONEDPL_HAS_RANGE_ALGORITHMS` in tests by the same way like our customers will check it in the code.

In this case we will see is it user-friendly way or no.

Reply:
> Why this static_assert has placed under #if TEST_UNNAMED_LAMBDAS check?

Actually, it is placed after the header. Since it is a library defined macro, you need to include some header first.

The customers will likely use it mainly as a condition in `#if` preprocessor directives.
I think `static_assert` is good enough as it will fail if either the macro is undefined (in which case its name will not be recognized as a known token and compilation will fail) or defined but not into a proper value (in which case the asserted condition will not be met).

Reply:
ok